src/corelibs/U2Formats/src/PDBFormat.cpp

```diff
@@ -280,12 +281,22 @@ void PDBFormat::PDBParser::parseMacromolecularContent(bool firstCompndLine, U2Op
     list
     Details
     */
+
+    // Returns the end-of-name index for `specification`.
+    auto endOfNameInd = [&](const QString &specification) {
+        int index = QRegExp(";\\s*$").indexIn(specification);
```

Comment:
Here you create a new reg-ex object on every invocation (may be in loop) which is anti-pattern
Normally you should re-use reg-ex objects. You can make is function static, or a class field.